hubblestack/fileclient.py

```diff
@@ -813,23 +775,15 @@ def get_file(self,
         if not dest2check:
             rel_path = self._check_proto(path)
 
-            log.debug(
-                'In saltenv \'%s\', looking at rel_path \'%s\' to resolve '
-                '\'%s\'', saltenv, rel_path, path
-            )
-            with self._cache_loc(
-                    rel_path, saltenv, cachedir=cachedir) as cache_dest:
+            log.debug("In saltenv '%s', looking at rel_path '%s' to resolve " "'%s'", saltenv, rel_path, path)
+            with self._cache_loc(rel_path, saltenv, cachedir=cachedir) as cache_dest:
                 dest2check = cache_dest
 
-        log.debug(
-            'In saltenv \'%s\', ** considering ** path \'%s\' to resolve '
-            '\'%s\'', saltenv, dest2check, path
-        )
+        log.debug("In saltenv '%s', ** considering ** path '%s' to resolve " "'%s'", saltenv, dest2check, path)
```

Comment:
Unnecessary " " in middle of log.debug.  Maybe change to: 
`log.debug("In saltenv '%s', ** considering ** path '%s' to resolve '%s'", saltenv, dest2check, path)`